graph/db/graph_test.go

```diff
@@ -75,7 +75,7 @@ func createLightningNode(priv *btcec.PrivateKey) *models.LightningNode {
 		AuthSigBytes:         testSig.Serialize(),
 		LastUpdate:           time.Unix(updateTime, 0),
 		Color:                color.RGBA{1, 2, 3, 0},
-		Alias:                "kek" + string(pub[:]),
+		Alias:                "kek" + hex.EncodeToString(pub),
```

Comment:
nit: IIUC Go strings UTF-8 by default, so we don't need anything specific. They're just an alias to `[]byte` hence a string containing UTF-8 runes can end up being longer byte length though.

Reply:
I don't think that applies here. Yes, if the byte slice contains a **valid** representation of UTF-8 characters, the string will look okay. But if you take random bytes (which the public key basically is), then you will get weird, non-printable characters in the SQL output. So I think this is required.

Reply:
(ah soz, only saw oli's comment after posting the above)

Reply:
Thanks for the explanation!